server/fishtest/util.py

```diff
@@ -444,18 +443,19 @@ def remaining_hours(run):
         if llr >= llr_bound:
             return 0
 
-        # Assume all tests use default book (UHO_4060_v3).
-        book_positions = 242201
-        t = scipy.stats.beta(1, 15).cdf(min(N / book_positions, 1.0))
-        expected_games_llr = int(2 * N * llr_bound / llr)
+        # Assume all tests use default book (UHO_Lichess_4852_v1).
```

Comment:
@vdbergh I updated the book/positions according to the default book for test, but I noticed that the regression test still uses the previous book. If I'm not wrong the update is still correct.